environment.yml

```diff
@@ -37,9 +37,29 @@ dependencies:
   - ncl
   - r-base
   # R packages needed for development
+  - r-abind
+  - r-akima<=0.6-2.3
```

Comment:
Please remember to add the pinning to https://github.com/conda-forge/esmvaltool-suite-feedstock/blob/46332d1f303b96679093b424fb0691ac39083e8b/recipe/meta.yaml#L147 when it comes to the release. Perhaps best to have an issue on the feedstock?